languages/fr/assess-api.json

```diff
@@ -68,15 +68,15 @@
     "greyOnGrey": "Gris sur gris clair",
     "whiteOnBlack": "Blanc sur noir",
     "customColorPalette": "Personnaliser le jeu de couleurs",
-    "fontSizeInstructions": "Ajuster la taille de police dans l/'activité",
+    "fontSizeInstructions": "Ajuster la taille de police dans l'activité",
     "small": "Petit",
     "normal": "Normal",
     "large": "Grand",
     "xlarge": "Très grand",
     "xxlarge": "Énorme",
     "zoomInstructions": "Vous pouvez effectuer un zoom avant ou arrière en utilisant les raccourcis clavier suivants:",
-    "command": "Cmd",
-    "control": "Ctrl",
+    "command": "Command",
```

Comment:
I was gonna say how about "Pomme" which I heard and read more than once, but then saw this
![image](https://user-images.githubusercontent.com/235000/67169207-cb9b4b00-f3f5-11e9-8564-5ff7385d90c9.png)

It's "Command**e**" as in "La commande"